### DISCUSSION TITLE
Possible to set up a relationship/reference field from backend?

### DISCUSSION THREAD

**Discussion by andregoldstein** [supabase/supabase#274] [Questions] [answered]
Hi there!

Just trying out supabase and was wondering if it's possible to add a reference field between two tables directly from the database view of the backend?

I tried adding a `clinic_id` field to a `clients` table - I used the `uuid` column type but I'm not sure if this is the right way to go to set up a relationship (or do you have to use the SQL tab directly?)

Thanks for any advice!

**Comment by kiwicopple** [comment, marked as answer]
Hey @andregoldstein thanks for dropping in. This will be available by the end of the month :) 

 For now you'll have to use the SQL Editor.

**Reply by andregoldstein** [reply]
Thanks very much for your reply! I'll give that a go 👍

**Reply by kiwicopple** [reply]
Let me know if you need any help!

**Reply by andregoldstein** [reply]
Thanks, discovering how non-existent are my Postgres skills!
